Tests/Core/ActivityIndicatorViewObserverTests.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  ActivityIndicatorViewObserverTests.swift
+//  Operations
+//
+//  Created by Matthew Holden 16/04/2016.
+//
+//
+
+import Foundation
+import XCTest
+@testable import Operations
+
+enum AnimationState {
+    case Animating
+    case NotAnimating
+}
+
+class TestableActivityIndicatorView: ActivityIndicatorViewAnimationInterface {
+    typealias IndicatorAnimationStateDidChangeBlock =  (toState: AnimationState) -> Void
```

Comment:
Nice. 😀 I'm a big fan of typealiasing closures like this, as you can probably guess from the codebase.